diff --git a/dist/src-CLLHiI8g.mjs b/dist/src-CLLHiI8g.mjs
index c000cb79467991dd55ef59cfaf74ca70a58c460c..46f8f37d647809bcf2c3436a4c55c2252aea10b7 100644
--- a/dist/src-CLLHiI8g.mjs
+++ b/dist/src-CLLHiI8g.mjs
@@ -2061,7 +2061,6 @@ const cleanUpOxlintConfig = (config) => {
 	}
 	if (!("files" in config)) {
 		cleanUpUselessOverridesEntries(config);
-		cleanUpDisabledRootRules(config);
 		cleanUpUnusedJsPlugins(config);
 	}
 };
